conftest.py

```diff
@@ -158,12 +158,14 @@ def host_at_least_8_3(host):
     version_str = "8.3"
     if not host.xcp_version >= version.parse(version_str):
         pytest.skip(f"This test requires an XCP-ng >= {version_str} host")
+    return host
```

Comment:
So, this fixture doesn't return anything on purpose, so that we use it outside function parameters (usually globally at the file level or the class level).

Maybe it's a good idea to have fixtures depend on this one, but I'm not sure. It hides the requirement from the test files.

Reply:
The requirements are also documented in the test initial comments and in the jobs description etc.
It simplifies some cases imho.

Reply:
What does it simplify?

Reply:
https://github.com/xcp-ng/xcp-ng-tests/pull/223/files#diff-006a1a28400f0825b0c1859d4cade24243e13ee0f16aa58fa072e57b08621d78R30

Reply:
I did check where it was used in the PR before adding my comments

Reply:
I don't think it's the right choice, but I've already given my arguments and if you think it's more important that a fixture can require this fixture that is made to skip tests, so be it.